platform/platform-app/src/main/resources/features.xml

```diff
@@ -981,6 +981,7 @@
         <feature prerequisite="true">platform-api</feature>
         <feature prerequisite="true">platform-security-session</feature>
         <feature prerequisite="true">platform-migrate-all</feature>
+        <bundle>mvn:ddf.platform/platform-logging/${project.version}</bundle>
```

Comment:
I didn't make a new feature as it didn't seem necessary since it is only one bundle. I did put it right after platform-api.